blog/app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,54 @@
+class ArticlesController < ApplicationController
+
+	http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
+
+	def index
+    	@articles = Article.all
+  	end
+
+  	def new
+		@article = Article.new
```

Comment:
very inconsistent tabbing in here. everything should be 2 spaced tab (just my preference but should at least be the same in the whole repo